MapboxMobileEvents/Categories/NSString+MMEVersions.m

```diff
@@ -46,6 +46,11 @@ - (NSString *)mme_semverBuildStringComponent;
 
 @implementation NSString (MMEVersions)
 
+- (NSString *)mme_stringByRemovingNonUserAgentTokenCharacters {
+return [[self componentsSeparatedByCharactersInSet:[NSCharacterSet characterSetWithCharactersInString:@" (),/:;<=>?@[]{}\"\\"]]
```

Comment:
I think spaces can be maintained "My Great Application" isn't an issue but "My App Build (123/123/123/;;;/.)" is